### PR TITLE
Regex to extract Dates from a given text

### PR DESCRIPTION
**Pull request by Savepeter2**
Added submissions for each tasks, input is a string and output is a list of strings

### CODE REVIEW

File: README.md
Comment (elseagle) [inline README.md:1]: Start a sentence with uppercase

File: Regex/regex_task1.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            Our hotline for urgent issues is +254711234567. Thank you for choosing our services. 
+    """
+    pass
+
+def extract_kenyan_no(text:str):
+    no_pattern = re.compile(r"\+254\d{9}")
+    extracted_nos = no_pattern.findall(text)
+    return extracted_nos
+
+test = kenyan_text.__doc__
```
Comment (elseagle) [inline Regex/regex_task1.py:26]: You can avoid using this approach that calls the __doc__ attribute by simply assigning the text to a variable as a multiline. 

The third Zen of Python is: "Simple is better than complex."

File: Regex/regex_task5.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    """
+
+def extract_uuid(text:str):
+    no_pattern = re.compile(r"[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}")
+    extracted_nos = no_pattern.findall(text)
+    return extracted_nos
+
+uuid_text = uuid_text.__doc__
+extracted_uuids = extract_uuid(uuid_text)
+print(extracted_uuids)
```
Comment (elseagle) [inline Regex/regex_task5.py:29]: Add a new line

**Review by elseagle — CHANGES_REQUESTED** [review]
Good work! However, the instruction was to attempt just one, let's stay within the confines of instructions for now. It's good you attempted all but for submission you are meant to follow the requirement.

Lastly, check out the in-line comments I gave

File: Regex/regex_task5.py (hunk 2 above)
Comment (Savepeter2) [reply Regex/regex_task5.py:29]: Sorry, I don't understand what you meant, add a new line to where

File: README.md
Comment (Savepeter2) [reply README.md:1]: This isn't clear to me also

File: Data_Wrangling/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+I preprocessed a dataset of over 500,000 rows, extracting features and cleaning the data through the following major steps:
```
Comment (elseagle) [inline Data_Wrangling/README.md:1]: No need to use first person while handling a README. I'll let it pass for this PR, but next time, avoid that.

**Review by elseagle — APPROVED** [review]

**Pull request closed without merging**